HelloTorchIOS/HelloTorchIOS/StrobeLights.swift

```diff
@@ -1,9 +1,39 @@
-//
-//  StrobeLights.swift
-//  HelloTorchIOS
-//
-//  Created by Luis Valle on 02/05/2018.
-//  Copyright © 2018 Novoda. All rights reserved.
-//
-
 import Foundation
+import AVFoundation
+
+class StrobeLights {
+    
+    let lightsController = LightsLogic()
+    var counter: Int = 0
+    var timer: Timer = Timer()
+    var frequency: Double = 500
+    var start = DispatchTime.now()
+    var end = DispatchTime.now()
+    var isCounting = false
```

Comment:
If this variables should not be changed outside of the logic of the class I would set them to private